bip-0360.mediawiki

```diff
@@ -11,696 +11,386 @@
   Type: Standards Track
   Created: 2024-12-18
   License: BSD-3-Clause
+  Requires: 340, 341 (partial), 342
 </pre>
 
-== Introduction ==
+==Introduction==
 
-=== Abstract ===
+===Abstract===
 
-This document proposes the introduction of a new output type, Pay to Quantum Resistant Hash (P2QRH), via a soft fork.
-P2QRH provides the same tapscript functionality as Pay to Taproot (P2TR) but removes the quantum-vulnerable
-key path spend in P2TR. By itself, P2QRH provides protection against long-exposure quantum attacks, 
-but requires PQ signatures to provide full security against Cryptographically Relevant Quantum Computers (CRQCs).
-P2QRH is designed to provide the foundation necessary for a future soft fork activating PQ signature verification 
-in tapscript.
+This document proposes a new output script type, Pay-to-Tapscript-Hash (P2TSH), via a soft fork. P2TSH outputs are Tapscript-native, hash-based outputs that disable the key-path spend functionality of a Pay-to-Taproot (P2TR) output by omitting the internal keys and the taproot 'tweak' step. The script-path spend functionality of P2TR is retained, and thus all spends of P2TSH outputs are Taproot-style script-path spends.
 
-=== Copyright ===
+Through this modification, P2TSH outputs provide a Tapscript-native, hashed output type that is:
```

Comment:
```suggestion
Through this modification, P2TSH outputs create the option of using Taproot in a manner that is:
```

Reply:
P2TSH is not Taproot, it is totally different than Taproot. There are just two similarities between P2TSH and Taproot: 1) both are Segwit output types and 2) locking scripts in both use Tapscript rather than Script.